examples/distributed/graphlearn_for_pytorch/partition_ogbn_dataset.py

```diff
@@ -0,0 +1,116 @@
+import argparse
+import ast
+import os.path as osp
+
+import graphlearn_torch as glt
+import torch
+from ogb.nodeproppred import PygNodePropPredDataset
+
+
+def partition_dataset(ogbn_dataset: str, root_dir: str, num_partitions: int,
+                      num_nbrs: glt.NumNeighbors, chunk_size: int,
+                      cache_ratio: float):
```

Comment:
Can we add documentation for this function?

Reply:
Hi, added documentation for the function here and also in README.md